### DISCUSSION TITLE
How can i disable dumps?

### DISCUSSION THREAD

**Discussion by StanBarrows** [beyondcode/herd-community#734] [Q&A] [answered]
- [ ] Is there a way to disable dumps? 

![CleanShot 2024-05-14 at 13 27 13@2x](https://github.com/beyondcode/herd-community/assets/10268813/f0174afc-db93-4402-9262-33e1a9ef79aa)

**Comment by mpociot** [comment, marked as answer]
Yeah, open the Dumps window and click on the antenna icon

![CleanShot 2024-05-14 at 14 13 49](https://github.com/beyondcode/herd-community/assets/804684/893de476-5376-4116-92a0-e6a58547bd7a)

**Reply by StanBarrows** [reply]
Thank you Marcel!

**Comment by Brugman** [comment]
@mpociot  That button toggles between "intercepting" and "paused". Is there a way to actually disable the feature? To turn off the green light?

![image](https://github.com/user-attachments/assets/984269ca-b64c-41a8-b048-8f94707b239b)

I need to use the `auto_prepend_file` setting in php.ini and Herd keeps overriding it with some code for Dumps.

Also, I don't have Pro so I would like to _not_ run services that I can't even use:

![image](https://github.com/user-attachments/assets/adec540c-9439-4828-8707-bc41415ee46d)
